src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,59 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        System.out.println("All cars:");
+        List<Car> cars = carService.getAll();
+        cars.forEach(System.out::println);
+        System.out.println(System.lineSeparator() + "Car with id 2:");
+        Car car = carService.get(2L);
+        System.out.println(car);
+        System.out.println(System.lineSeparator() + "Add new car:");
+        Manufacturer manufacturer = new Manufacturer("Peugeot", "France");
+        manufacturer.setId(25L);
+        car = new Car("208", manufacturer);
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        Driver driver = driverService.get(3L);
+        Driver driver2 = driverService.get(5L);
+        List<Driver> drivers = new ArrayList<>();
+        drivers.add(driver);
+        drivers.add(driver2);
+        car.setDrivers(drivers);
+        System.out.println(carService.create(car));
+        System.out.println(System.lineSeparator() + "Delete car with id 4:");
+        System.out.println(carService.delete(4L));
+        System.out.println(System.lineSeparator() + "Update car with id 3:");
+        manufacturer = new Manufacturer("Lexus", "Japan");
+        manufacturer.setId(5L);
+        Car car2 = new Car("NX", manufacturer);
+        car2.setId(3L);
```

Comment:
the same here, I suppose it would be better to use create() method to create a car and then just set a new value to some of its field and update the data